lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -317,7 +317,7 @@ def debug_message(cause = nil, info_hash = {})
     def fetch_events(kubectl)
       return {} unless exists?
       out, _err, st = kubectl.run("get", "events", "--output=go-template=#{Event.go_template_for(type, name)}",
-        log_failure: false)
+        log_failure: false, use_namespace: !global?)
```

Comment:
`kubectl api-resouces` indicates that events are `NAMESPACED`. Running `kubectl get events` returns events about `Nodes` which are also global, so I believe this is the correct behavior.